sub_peers.js

```diff
@@ -35,6 +35,7 @@ function TChannelSubPeers(channel, options) {
     var self = this;
     this.peerScoreThreshold = this.options.peerScoreThreshold || 0;
     this.choosePeerWithHeap = channel.choosePeerWithHeap;
+    this.socketInitTimeout = channel.initTimeout;
```

Comment:
per offline discussion with jake, this channel is a sub channel, and points to the initTimeout provided by client.